cpp/velox/compute/WholeStageResultIterator.cc

```diff
@@ -101,9 +101,44 @@ std::shared_ptr<ColumnarBatch> WholeStageResultIterator::next() {
 }
 
 int64_t WholeStageResultIterator::spillFixedSize(int64_t size) {
+  std::string poolName{pool_->root()->name() + "/" + pool_->name()};
+  std::string logPrefix{"Spill[" + poolName + "]: "};
+  LOG(INFO) << logPrefix << "Trying to reclaim " << size << " bytes of data...";
+  LOG(INFO) << logPrefix << "Pool has reserved " << pool_->currentBytes() << "/" << pool_->root()->reservedBytes()
+            << "/" << pool_->root()->capacity() << "/" << pool_->root()->maxCapacity() << " bytes.";
+  LOG(INFO) << logPrefix << "Shrinking...";
+  int64_t shrunk = pool_->shrinkManaged(pool_.get(), size);
+  LOG(INFO) << logPrefix << shrunk << " bytes released from shrinking.";
```

Comment:
@zhztheplayer Should we remove this log, or change log level.

Reply:
Thanks for the suggestion.

I think we can at least use debug for shrinking log ?

Or for reclaiming (spilling) log as well?